pkg/io/commonACL.go

```diff
@@ -3,66 +3,32 @@ Copyright 2023- IBM Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package csvio
+package io
 
 import (
 	"errors"
 	"fmt"
 	"strconv"
 
-	"github.com/np-guard/models/pkg/interval"
 	"github.com/np-guard/models/pkg/netp"
 	"github.com/np-guard/models/pkg/netset"
-
 	"github.com/np-guard/vpc-network-config-synthesis/pkg/ir"
 )
 
-// Write prints an entire collection of acls as a single CSV table.
-func (w *Writer) WriteACL(collection *ir.ACLCollection, vpc string) error {
-	if err := w.w.WriteAll(aclHeader()); err != nil {
-		return err
-	}
+func WriteACL(collection *ir.ACLCollection, vpc string) ([][]string, error) {
+	res := make([][]string, 0)
 	for _, subnet := range collection.SortedACLSubnets(vpc) {
 		vpcName := ir.VpcFromScopedResource(subnet)
 		aclTable, err := makeACLTable(collection.ACLs[vpcName][subnet], subnet)
-		if err != nil {
-			return err
-		}
-		if err := w.w.WriteAll(aclTable); err != nil {
-			return err
-		}
-	}
-	return nil
-}
-
-func makeACLTable(t *ir.ACL, subnet string) ([][]string, error) {
-	rules := t.Rules()
-	rows := make([][]string, len(rules))
-	for i, rule := range rules {
-		aclRow, err := makeACLRow(i+1, rule, t.Name(), subnet)
 		if err != nil {
 			return nil, err
 		}
-		rows[i] = aclRow
-	}
-	return rows, nil
-}
-
-func aclPort(p interval.Interval) string {
-	if p.Equal(netp.AllPorts()) {
-		return "any port" //nolint:goconst // independent decision for SG and ACL
+		res = append(res, aclTable...)
 	}
-	return fmt.Sprintf("ports %v-%v", p.Start(), p.End())
+	return res, nil
 }
 
-func action(a ir.Action) string {
-	if a == ir.Deny {
-		return "Deny"
-	}
-	return "Allow"
-}
-
-func aclHeader() [][]string {
+func ACLHeader() [][]string {
```

Comment:
```suggestion
func makeACLHeader() [][]string {
```
Doesn't have to be exported